### DISCUSSION TITLE
posts/58153/

### DISCUSSION THREAD

**Discussion by giscus** [STEVE-1936550490/blog_comment#12] [Announcements]
# posts/58153/

ふなびん（船便）ーーーーー海运
 しょるい（書類）ーーーーー文件、文稿
 げんこう（原稿）ーーーーー稿子、原稿
 きじ（記事）ーーーーー报道
 メモ　ーーーーー记录
 えきまえ（駅前）ーーーーー车站一带
 はし（橋）ーーーーー桥
 かど（角）ーーーーー拐角
 おうだんほどう（横断歩道）ーーーーー人行横道
 みぎ（右）ーーーーー右
 ひだり（左）ーーーーー左
 こうさてん（交差点）ーーーーー十字路口

https://steve-1936550490.github.io/posts/58153/

**Comment by STEVE-1936550490** [comment]
上班摸鱼是吧
